NBitcoin/Transaction.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace NBitcoin
 {
-	public class OutPoint : IBitcoinSerializable
+	public class OutPoint : IBitcoinSerializable, IComparable, IComparable<OutPoint>, IEquatable<OutPoint>

```

Comment:
`IComparable` makes no sense for `OutPoint` outpoint, unless you want to implement @kristovatla's BIP69: https://github.com/bitcoin/bips/blob/master/bip-0069.mediawiki

Reply:
I implemented these interface just for use of convenience on my code. (To use in F# collection types) And the actual logic does not really matter. So yes, I think modifying these methods to follow BIP69 is the right way to go.